src/style.css

```diff
@@ -2,2 +1,82 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to include fallback fonts in case the main one doesn't load. For example, 'font-family: Roboto, Arial, sans-serif;'.